_data/sites.json

```diff
@@ -983,6 +993,16 @@
             "burst.net"
         ]
     },
+    
+    {
+        "name": "Buycraft",
+        "url": "https://server.tebex.io/profile/delete",
+        "difficulty": "easy",
+        "notes": "Buycraft was moved to Tebex, just click the 'delete profile' button in your account menu",
+        "domains": [
+            "tebex.io"
+        ]
+    },
```

Comment:
Needs to come after `Buycott` to keep sorted

Reply:
I'm sorry, seems the fever made me forget my ABCs. Should be fixed now

Reply:
It's alright :)